packages/react-native/template/android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         compileSdkVersion = 33
         targetSdkVersion = 33
 
-        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
+        // We use NDK 23 which has both Apple Silicon support and is the side-by-side NDK version from AGP.
```

Comment:
Can you undo this change?
This is our template and it will create more noise than anything

Reply:
Thank you for the review! Addressed with [955f8de](https://github.com/facebook/react-native/pull/38344/commits/955f8dea1f4705aa9ffdd10105c18bb55b7a8ec0)